rclcpp/test/rclcpp/executors/test_multi_threaded_executor.cpp

```diff
@@ -96,4 +96,48 @@ TEST_F(TestMultiThreadedExecutor, timer_over_take) {
   auto timer = node->create_wall_timer(PERIOD_MS, timer_callback, cbg);
   executor.add_node(node);
   executor.spin();
+
+}
+
+TEST_F(TestMultiThreadedExecutor, catch_exception) {
```

Comment:
can this test be moved to the "common executors tests" ?

Reply:
I'll look into it...

Reply:
done